Maple2.Server.Game/Packets/GroupChatPacket.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Game.GroupChat;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class GroupChatPacket {
+    private enum Command : byte {
+        Load = 0,
+        Create = 1,
+        Invite = 2,
+        Join = 3,
+        Leave = 4,
+        AddMember = 6,
+        RemoveMember = 7,
+        LoginNotice = 8,
+        LogoutNotice = 9,
+        Chat = 10,
+        Error = 13,
+    }
+
+    public static ByteWriter Load(GroupChat groupChat) {
+        var pWriter = Packet.Of(SendOp.GroupChat);
+        pWriter.Write<Command>(Command.Load);
+        pWriter.WriteInt(groupChat.Id);
+        pWriter.WriteByte((byte) groupChat.Members.Count);
+        foreach ((long characterId, GroupChatMember member) in groupChat.Members) {
+            pWriter.WriteByte(1); // Login notification ?
```

Comment:
should this be a bool then? Also if that's the case does it mean you get spammed with "login" for each member of the chat whenever it loads?

---

Seems like this is probably the same byte as `AddMember` maybe it just means there IS a member.